tests/phpunit/elementor/modules/atomic-widgets/test-atomic-widget-base.php

```diff
@@ -73,6 +73,83 @@ public function test_get_atomic_settings__returns_only_settings_that_are_defined
 		], $settings );
 	}
 
+	public function test_get_atomic_settings__return_transformed_value() {
+		// Arrange.
+		$widget = $this->make_mock_widget( [
+			'props_schema' => [
+				'should_transform' => Atomic_Prop::make(),
+			],
+			'settings' => [
+				'should_transform' => [
+					'$$type' => 'classes',
+					'value' => [ 'one', 'two', 'three' ],
+				],
+			],
+		] );
+
+		// Act.
+		$settings = $widget->get_atomic_settings();
+
+		// Assert.
+		$this->assertEquals( [
+			'should_transform' => 'one two three',
+		], $settings );
+	}
+
+	public function test_get_atomic_settings__return_null_when_transformer_not_exits() {
```

Comment:
```suggestion
	public function test_get_atomic_settings__return_null_when_transformer_does_not_exist() {
```

learn englishhh

Reply:
It's a test description that should be short and explanatory.
but changed